content/en/containers/cluster_agent/clusterchecks.md

```diff
@@ -155,6 +155,26 @@ clusterAgent:
 ```
 
 **Note**: This is separate from the `datadog.confd` section, where the files are created in the node-based Agents. The `<INTEGRATION_NAME>` must exactly match the desired integration check you want to run.
+
+{{% /tab %}}
+{{% tab "Operator" %}}
+With the Datadog Operator, these configuration files can be created within the `spec.override.clusterAgent.extraConfd.configDataMap` section:
```

Comment:
Could you add an example for `spec.override.clusterAgent.extraConfd.configMap` as well? There's an example for how the `configMap` section can be used in this PR: https://github.com/DataDog/datadog-operator/pull/616